src/genie/libs/parser/iosxe/show_wireless.py

```diff
@@ -2053,3 +2053,761 @@ def cli(self, output=None):
+          "authentication_key_management": str,
+          "user_defined_private_network": str,
+          "user_defined_private_network_drop_unicast": str,
+          "encrypted_traffic_analytics": str,
+          "protected_management_frame__802.11w": str,
+          "eap_type": str,
+          "vlan_override_after_webauth": str,
+          "vlan": str,
+          "multicast_vlan": int,
+          "wifi_direct_capabilities": {
+              "wifi_direct_capable": str
+          },
+          "central_nat": str,
+          "session_manager": {
+              "point_of_attachment": str,
+              "iif_id": str,
+              "authorized": str,
+              "session_timeout": int,
+              "common_session_id": str,
+              "acct_session_id": str,
+              "last_tried_aaa_server_details": {
+                  "server_ip": str
+              },
+              "auth_method_status_list": {
+                  "method": str,
+                  "sm_state": str,
+                  "sm_bend_state": str
+              },
+              "local_policies": {
+                  "service_template": str,
+                  "vlan_group": str,
+                  "absolute_timer": int
+              },
+              "server_policies": {
+                  "output_sgt": str
+              },
+              "resultant_policies": {
+                  "output_sgt": str,
+                  "vlan_name": str,
+                  "vlan_group": str,
+                  "vlan": int,
+                  "absolute_timer": int
+              }
+          },
+          "dns_snooped_ipv4_addresses": str,
+          "dns_snooped_ipv6_addresses": str,
+          "client_capabilities": {
+              "cf_pollable": str,
+              "cf_poll_request": str,
+              "short_preamble": str,
+              "pbcc": str,
+              "channel_agility": str,
+              "listen_interval": int
+          },
+          "fast_bss_transition_details": {
+              "reassociation_timeout": int
+          },
+          "11v_bss_transition": str,
+          "11v_dms_capable": str,
+          "qos_map_capable": str,
+          "flexconnect_data_switching": str,
+          "flexconnect_dhcp_status": str,
+          "flexconnect_authentication": str,
+          "flexconnect_central_association": str,
+          "client_statistics": {
+              "number_of_bytes_received": int,
+              "number_of_bytes_sent": int,
+              "number_of_packets_received": int,
+              "number_of_packets_sent": int,
+              "number_of_policy_errors": int,
+              "radio_signal_strength_indicator_dbm": str,
+              "signal_to_noise_ration_db": int
+          },
+          "fabric_status": "Disabled",
+          "radio_measurement_enabled_capabilities": {
+              "capabilities": list
+          },
+          "client_scan_report": str,
+          "nearby_ap_statistics": {
+              "ap_names": dict
+          },
```

Comment:
better to mention more instead of `dict`. and schema might be like below because not sure if maximum number of antenna is always 2.

Below example with values. 
```suggestion
          "nearby_ap_statistics": {
              "ap_names": {
                  "prateekk_cos_1 (slot 1)": {
                      "antenna": {
                           "antenna 0": {
                               "seconds_ago": 13,
                               "dBm": -25
                           },
                           "antenna 1": {
                               "seconds_ago": 13,
                               "dbm": -25
                           }
                       }
                   }
               }
          },
```
sample output for above:
```
Nearby AP Statistics:
  prateekk_cos_1 (slot 1)
  	antenna 0: 13 s ago	........ -25  dBm
  	antenna 1: 13 s ago	........ -25  dBm

Reply:
Updated.